cmd/ocm-backplane/login/login.go

```diff
@@ -520,6 +520,11 @@ func doLogin(api, clusterID, accessToken string) (string, error) {
 		return "", err
 	}
 
+	err = backplaneapi.CheckResponseDeprecation(resp)
+	if errors.Is(err, backplaneapi.ErrDeprecation) {
+		logger.Warnf("The server indicated that this version of backplane-cli is deprecated, please update as soon as possible.")
```

Comment:
Do you think it's better to include the current version of `bp-cli` to make it more actionable? e.g
```
logger.Warnf("The server indicated that backplane-cli version %s is deprecated. Please update as soon as possible.", currentVersion)
```